src/cider/nrepl/middleware/slurp.clj

```diff
@@ -0,0 +1,101 @@
+(ns cider.nrepl.middleware.slurp
```

Comment:
I think this should be an `util` namespace, as it's not really a middleware itself.